index.html

```diff
@@ -1191,6 +1214,37 @@ <h4>Rate-limiting change notifications</h4>
           counters, if not accessed via a higher level framework.
         </p>
       </section>
+      <section>
+        <h4>Rate obfuscation</h4>
+        <p>
+          The specification will recommend the implementation to keep track of the number of
+          pressure changes over an implementation-specific sliding observation window and
+          set a flag if an implementation-specific threshold for the number of pressure
+          changes is exceeded. Similarly, it is also recommended for the implementation to
+          observe any abnormal activity such as a high number of pressure state changes
+          spanning across multiple states, and set this flag similarly.
+        </p>
+        <p>
+          If this flag is set, the implementation is recommended to give the pressure observer
+          a penalty during which it will not be able to inform scripts of changes in its
+          pressure state as it normally would. The duration of this penalty is
+          implementation-defined and it is recommended to be randomized.
+          When [=notify pressure observers=] resumes operation after the penalty, it only
+          reports the latest pressure state and disregards any interim state information
+          received from the platform collector during this penalty.
+        </p>
+      </section>
+      <section>
+        <h4>Break calibration</h4>
+        <p>
+          In a calibration process an attacker tries to manipulate the CPU so that this
+          API would report a transition into a certain pressure state in response to the
+          pressure exerted by the fabricated workload. By slightly changing at runtime
```

Comment:
Maybe say that they will find to find the workload pressure that with the highest probability will result in a certain state?

Reply:
Fixed in https://github.com/w3c/compute-pressure/pull/219/commits/3cfdf72b212348c67ab3d2cb4b9f4716b502c96d